publications-2019.bib

```diff
@@ -232,4 +232,14 @@ @article{Cheng2019
   journal = {Computer Modeling in Engineering {\&} Sciences}
 }
 
+
+
+@Article{LZ19,
+  author =       {Z. Luo and Y. F. Zhao},
```

Comment:
I don't have access to this paper, but in the 'cite as' category I just see 'Y Zhao'. Is it 'Y F Zhao' in the paper?